syntax = "proto3";

import "task.proto";

package task;

message WorkerTaskId {
    string identity = 1;
    TaskId id = 2;
}

message WorkerMessage {
    oneof data {
        uint32 core_count = 4; 
        WorkerTaskId task_id = 5;
    }
}

